_build/templates/default/sass/_navbar.scss

```diff
@@ -188,6 +188,18 @@
         &:after {
           border-right-color: $subnavBgHover;
         }
+      }      
+      &.sub {
+      &:after {
+        @extend %pseudo-font;
+        position: absolute;
+        color: $subnavDescriptionColor;
+        content: fa-content($fa-var-caret-right);
```

Comment:
```suggestion
        content: $fa-var-caret-right;
```

Reply:
fa-content is correct here, it's a new function from the updated font-awesome

Reply:
@Alroniks This is probably worth changing in the future, because the old variant is found here and in other scss files.